packages/workflows/src/cli/list/activeWorkflows/ActiveWorkflows.definition.ts

```diff
@@ -0,0 +1,52 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { ICommandDefinition } from "@brightside/imperative";
+import { ListOptions } from "./ActiveWorkflows.options";
+import { join } from "path";
+
+
+/**
+ * This object defines the command for listing workflow instance(s) in zOSMF.
+ * This is not something that is intended to be used outside of this npm package.
+ *
+ * @private
+ */
+export const ActiveWorkflows: ICommandDefinition = {
+    name: "active-workflows",
+    aliases: ["acw"],
+    description: "List workflow instance(s) in z/OSMF.\n" +
```

Comment:
maybe we can use summary here ? Like this : 
`summary: "List workflow instance(s) in z/OSMF.",`

Currently if you do:
`bright wf list -h `

the output is : 
```
   active-workflows | acw       List workflow instance(s) in z/OSMF.
                                Omitting all options will list all workflows on
                                the sysplex

```

And with the summary it will be : 
```
   active-workflows | acw       List workflow instance(s) in z/OSMF.

```

Reply:
List **active** workflow instance(s) in z/OSMF. :stuck_out_tongue_closed_eyes: